core/dbt/config/profile.py

```diff
@@ -109,6 +111,7 @@ def __init__(
         self.threads = threads
         self.credentials = credentials
         self.profile_env_vars = {}  # never available on init
+        self.manage_schemas = manage_schemas
```

Comment:
Hm. I'm not convinced this makes sense as an attribute directly within the connection profile, versus a [global config](https://docs.getdbt.com/reference/global-configs) — which could still be set in the `config` block of `profiles.yml` (default when running on this "machine"), but also as an env var and CLI flag. It wouldn't be tied to `target` in quite the same way, but dev/prod behavior would still be achievable in the same file or via env var, and it would be more consistent with how other runtime configs work.

(I'm overdue for a broader issue/discussion about the next phase of config reconciliation—it's better than it used to be, but there are definitely still some divergent patterns today)

Reply:
The idea was indeed to be able to specify different behavior for dev/pro discrepancies, or other use cases. If we can achieve the same with the global config, that could be an option too. @bneijt what do you think?

How might the config reconciliation discussion influence this?

Reply:
@jtcohen6 I do see the problems with config (patterns differ for profiles and project.yml for example), but defining it per target seemed like the best location for the config as all targets would mean that you can't have a managed testing database while leaving dev unmanaged (because everybody is experimenting there). But if we move the whole logic to a new command, than this would nolonger be an issue and we can just drop this config.

Reply:
I think we're on the same page that the goal here is conditional opt-in for schema management, since users likely want to manage schemas for some runtime environments (= prod) but not for others (= dev).

We've tended to achieve that via flags / environment variables, rather than additional `target` values in `profiles.yml`. A totally separate command, or a hook/operation which users could set up themselves, would achieve the same thing more explicitly.